doc/user-guide.adoc

```diff
@@ -477,6 +478,17 @@ Users can turn off the auto-detection mechanism by setting the `bindings.autoDet
 
 _This feature is only available if you have Service Binding Operator installed on your cluster._
 
+==== Exposing `RuntimeComponent` applications as Provisioned Services
+
+A `RuntimeComponent` application can be configured to behave as a link:++https://github.com/k8s-service-bindings/spec#provisioned-service++[Provisioned Service] defined by the link:++https://github.com/k8s-service-bindings/spec++[Service Binding Specification].
+
+According to the specification, a Provisioned Service resource must define a `.status.binding.name` which is a `LocalObjectReference`-able to a Secret.
+To expose your application as a Provisioned Service, set the `.spec.bindings.expose.enabled` parameter to a value of `true`. The Runtime Component Operator creates a *binding secret* named `<CR_NAME>-expose-binding` and adds the following entries to the secret: `host`, `port`, `protocol`, `basePath` and `uri`.
+
+To override the default values for the entries in the binding secret or to add new entries to the secret, create an *override secret* named `<CR_NAME>-expose-binding-override` and add any entries to the secret. The operator reads the content of the override secret and overrides the default values in the binding secret.
+
+Once a `RuntimeComponent` application is exposed as a Provisioned Service, a `ServiceBindingRequest` can refer to the application as a backing service.
```

Comment:
For `ServiceBindingRequest`, either put a noun after it our replace it with English words.
Example: Say: ".... a service binding request can refer...." or " ... a `ServiceBindingRequest` <nooun here> can refer to ... ".